### ISSUE TITLE
Is it possible to see it in marketplace?

### ISSUE THREAD

**Issue by amstiel** [mallowigi/Custom-Syntax-Highlighter#2]
Sup, awesome syntax highlighting addition. Is it possible to see your plugin in JetBrains marketplace someday?

**Comment by leo4all** [comment]
+1

**Comment by mallowigi** [comment]
This repository is some kind of PoC, not a real plugin per se... 

I've already implemented language additions in the Material Theme UI plugin, and others have done the same

- https://plugins.jetbrains.com/plugin/13592-php-syntax-color-highlighter
- https://plugins.jetbrains.com/plugin/13979-kotlin-additional-highlighting

**Comment by amstiel** [comment]
The plugins you gave as an example add additions only for certain languages. But this plugin is a kind of compilation in which, among other things, the user can choose colors himself.

I'm big fan of Material Theme UI plugin, I used to use it myself, but now a combination of Atom Material Icons, Custom Syntax Highlighter and VSCode Dark Plus Theme is enough for me.

Now I'm using Custom Syntax Highlighter for more detailed syntax highlighting, as in VSCode (especially JS, PHP). It's uncomfortable to build plugin myself, then install manually  🙃 

So, maybe there is a chance to see it in marketplace? 🙂
I understand, this is PoC, but it works fine and it's already part of Material Theme UI. Atom Material Icons is part of Material Theme UI, btw.

**Comment by mallowigi** [comment]
I'm not sure I follow. Do you mean to release as a standalone plugin adding additions to a bunch of languages in the same flavor as I implemented in the Material UI plugin? Or do you mean a plugin where users can "plug-in" their own additions ?

**Comment by amstiel** [comment]
Yeah, standalone plugin which provide implemented additions, like in Material UI.
I hope that there will be more supported languages in future. Most likely I will help with this, as I helped with PHP Additions 😁
